doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+================================================
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+Prerequisites
+=============
+
+Before reading this tutorial you should know a bit of Python. If you
+would like to refresh your memory, take a look at the `Python
+tutorial <https://docs.python.org/tutorial/>`__.
+
+If you wish to work the examples in this tutorial, you must also have
+some software installed on your computer. Please see
+https://scipy.org/install.html for instructions.
+
+Learner profile
+===============
+
+This tutorial is aimed at people who have some basic prior knowledge of linear
+algebra and arrays in NumPy and want to understand how n-dimensional
+(:math:`n>=2`) arrays are represented and can be manipulated. In particular, if
+you don't know how to apply common functions to n-dimensional arrays in stacked
+format (without using for-loops), or if you wish to understand axis and shape
+properties for n-dimensional arrays, this tutorial might be of help.
+
+Learning Objectives
+===================
+
+After this tutorial, you should be able to:
+
+- Understand the difference between one-, two- and n-dimensional arrays in NumPy;
+- Understand how to apply some linear algebra operations in stacked format;
+- Understand axis and shape properties for n-dimensional arrays.
+
+Content
+=======
+
+In this tutorial, we will use linear algebra to generate an approximation of an
+image. We'll use the ``face`` image from the `scipy.misc` module:
+
+    >>> from scipy import misc
+    >>> img = misc.face()
+
+If you would rather use your own image, in order to transform this image into a
+numpy array that can be manipulated, we will use the ``imread`` function from the
+`matplotlib.image` submodule.
+
+Now, ``img`` is a numpy array, as we can see when using the ``type`` function:
+
+::
+
+    >>> type(img)
+    numpy.ndarray
+
+We can see the image using the `matplotlib.pyplot.imshow` function:
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.imshow(img)
+
+.. image:: https://docs.scipy.org/doc/scipy/reference/_images/scipy-misc-face-1.png
```

Comment:
You want to use `.. plot::` above the two lines of code so the output gets rendered here when docs get built.

Reply:
This doesn't work since the code is mixed with text. At this point `img` has been defined at another code block so I get an error when building the docs. Would it be better to use a separate script for each image and use something like `.. plot:: gallery/plot_face.py`? Note that I didn't include images for the next steps exactly for the same reason.

Reply:
yes indeed, script file would be good. there's examples of that in `doc/source/user/plots/`